### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by GustavoFSProg**
Fiz o layout basico do prifile, e ainda não fiz a funcionalidades de atualização.

pic

### CODE REVIEW

File: src/components/Label/styles.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const StyledLabel = styled.label`
   font-weight: ${props => props.theme.typography.weight.bold};
   padding-left: 3px;
   padding-right: 13px;
+  color: 'green';
```
Comment (roavellarm) [inline src/components/Label/styles.js]: Why did you change this?

File: src/components/Text/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  paddingRight: PropTypes.string,
+}
+
+Text.defaultProps = {
+  fontSize: '15px',
+  fontWeight: undefined,
+  paddingLeft: undefined,
+  paddingRight: undefined,
+}
+export default Text
```
Comment (roavellarm) [inline src/components/Text/index.js]: Great! But there already was a text component called Typography. You could use it instead of creating this new similar component

File: src/components/Text/styles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+const StyledText = styled.text`
+  font-size: ${props => `${props.theme.typography.size.md}px`};
+  font-weight: ${props => props.theme.typography.weight.bold};
+  padding-left: 3px;
+  padding-right: 13px;
+  color: ${color => color || 'orange'};
+`
+
+export default StyledText
```
Comment (roavellarm) [inline src/components/Text/styles.js]: There is no need to use export default in this case. Usually `styles.js` files will have more than one styled-components  exports. So usually is better to export them like
```JS
export const StyledText = styled.text`
  font-size: ${props => `${props.theme.typography.size.md}px`};
  font-weight: ${props => props.theme.typography.weight.bold};
  padding-left: 3px;
  padding-right: 13px;
  color: ${color => color || 'orange'};
`
```

File: src/pages/Profile/styles.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+export const StyledNumber = styled.div`
+  display: flex;
+  height: 20px;
+  width: 20px;
+  background: #32a852;
+  padding: 4px;
+  border-radius: 100%;
+  justify-content: center;
+  align-items: center;
+`
```
Comment (roavellarm) [inline src/pages/Profile/styles.js]: You can remove this file since you are not using it.

File: src/components/Label/styles.js (hunk 1 above)
Comment (GustavoFSProg) [reply src/components/Label/styles.js]: In fact I cahge for testng and I forgot do return the value,

File: src/pages/Profile/styles.js (hunk 4 above)
Comment (GustavoFSProg) [reply src/pages/Profile/styles.js]: I removed this file now

File: src/components/Text/styles.js (hunk 3 above)
Comment (GustavoFSProg) [reply src/components/Text/styles.js]: ok, I understand it

File: src/pages/Profile/index.js [hunk 5]
```diff
@@ -1,7 +1,45 @@
...
+        </Column>
+        <Column
+          margin="10px"
+          size={12}
+          justifyContent="center"
+          background="#302934"
+        >
+          <Typography>
+            Nome:
+            {profile.name}
```
Comment (roavellarm) [inline src/pages/Profile/index.js]: Try adding space after colon, like:
```JS
{`Nome: ${profile.name}`}
```

File: src/components/Label/styles.js (hunk 1 above)
Comment (roavellarm) [reply src/components/Label/styles.js]: Great! Just remove it in this case, please!

File: src/pages/Profile/index.js (hunk 5 above)
Comment (GustavoFSProg) [reply src/pages/Profile/index.js]: Ok

File: src/components/Avatar/index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  src: PropType.string.isRequired,
+  size: PropType.string,
+  border: PropType.string,
+}
+
+Avatar.defaultProps = {
+  size: undefined,
+  border: undefined,
+}
+export default Avatar
```
Comment (roavellarm) [inline src/components/Avatar/index.js]: Should be an empty line between 18 and 19. Maybe we should install husky to avoid pushing to github linter issues

Comment (GustavoFSProg) [reply src/components/Avatar/index.js]: We could install husky,  but i  now that, but i passed throw without see that

Comment (roavellarm) [reply src/components/Avatar/index.js]: You can run `yarn lint` to check if there is some linter issue before commits also

**Review by roavellarm — APPROVED** [review]
Great job!
